### PR TITLE
Add an average_winrate_observed metric to tensorboard

### PR DESCRIPTION
**Pull request by amj**
put something on tensorboard of the ratio of w/b wins in the train data.

### CODE REVIEW

File: dual_net.py [hunk 1]
```diff
@@ -541,10 +546,18 @@ def get_estimator():
 def _get_nontpu_estimator():
     session_config = tf.ConfigProto()
     session_config.gpu_options.allow_growth = True
-    run_config = tf.estimator.RunConfig(
-        save_summary_steps=FLAGS.summary_steps,
-        keep_checkpoint_max=FLAGS.keep_checkpoint_max,
-        session_config=session_config)
+    if FLAGS.num_gpus > 1:
```
Comment (sethtroisi) [inline dual_net.py]: can you extract this num_gpus change to a separate PR?

Comment (amj) [reply dual_net.py]: whups, sorry.  Sure can.

Comment (amj) [reply dual_net.py]: (it doesn't work)
